src/web/core/src/main/resources/GeoServerApplication.properties

```diff
@@ -347,8 +350,13 @@ GlobalSettingsPage.verboseExceptions = Include stack trace in service exception
 GlobalSettingsPage.verboseMessaging  = Verbose XML output (pretty print) 
 GlobalSettingsPage.trailingSlashMatch = Match URLs with trailing slash (Requires restart)
 GlobalSettingsPage.xmlPostRequestLogBufferSize  = Number of characters to log for incoming POST requests (0 to disable)
-GlobalSettingsPage.webUIMode = WebUI Mode
 GlobalSettingsPage.webUISettings = WebUI Settings
```

Comment:
There is an inconsistency between the documentation heading you added above, the actual heading in the user interface, and the name used in the existing documentation:

GeoServerApplication.properties: WebUI Settings
globalsettings.rst: UI Display Settings
name: Web Administration Interface

See docs:

- https://docs.geoserver.org/latest/en/user/webadmin/index.html
- https://docs.geoserver.org/latest/en/user/gettingstarted/web-admin-quickstart/index.html

I recommend changing this heading to "Web Administration Interface"

Reply:
Actually, the “WebUI Settings” section controls the Web UI redirect behavior.
My addition is placed below it as “UI Display Settings”.

Nonetheless, I’m renaming it to “Web Administration Interface Settings”, following your recommendation.